include/pokemon.h

```diff
@@ -407,6 +407,11 @@ struct SpindaSpot
     u16 image[16];
 };
 
+struct __attribute__((packed)) LevelUpMove {
+    u16 move:9;
+    u16 level:7;
+};
+
```

Comment:
dead code. i take it this doesn't match? i think you have to specify packed in the array declaration if that's giving you trouble.